### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by Aissaoui-Ahmed**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import './App.css';
+import './assets/css/app.css';
```
Comment (atefBB) [inline src/App.tsx:1]: I've fixed this but didn't push it yet !
Thx

File: public/manifest.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
       "sizes": "512x512"
     }
   ],
-  "start_url": ".",
+  "start_url": "./quran-react-app",
```
Comment (atefBB) [inline public/manifest.json]: Why should we edit this?

Comment (Aissaoui-Ahmed) [reply public/manifest.json]: Link of pictures is broken 
should be https://atefbb.github.io/quran-react-app/assets/images/quran/%3Cpic_number%3E.png
image

Comment (atefBB) [reply public/manifest.json]: Thank you, my friend! jazak Allah khayran !

**Review by atefBB — APPROVED** [review]

**Pull request closed without merging**